src/bifrost/core/config.rs

```diff
@@ -63,7 +63,11 @@ impl Default for Config {
         // Do not let users construct a Bifrost realm from within _THE_ Bifrost
```

Comment:
However, a there's nothing wrong with a user running `bifrost setup` from their home directory. This should be fixed.